app/models/carrier.rb

```diff
@@ -0,0 +1,33 @@
+require 'active_shipping'
+
+class Carrier < ActiveRecord::Base
+
+  ORIGIN = ActiveShipping::Location.new(country: 'US', state: 'WA', city: 'Seattle', zip: '98161')
+
+  def self.estimate_usps_shipping(items, state, city, zip)
+
+    # parsed_request = JSON.parse(request)
```

Comment:
be sure to delete these lines that you don't need anymore before submitting a PR